Framework/include/RunTopTagger.h

```diff
@@ -439,7 +439,11 @@ class RunTopTagger
 
     void operator()(NTupleReader& tr)
     {
-        runTopTagger(tr);
+        const auto& lostCauseEvent = tr.getVar<bool>("lostCauseEvent" + myVarSuffix_);
+        const auto& fastMode       = tr.getVar<bool>("fastMode");
+
+        if (!lostCauseEvent or !fastMode)
```

Comment:
The problem with doing it in `Config.h` is at that point, there's no knowledge of anything about an event to make a decision about. And as mentioned previously, the modules should all be instantiated, since in a given running of an analyzer, there will always be some events worth processing to the very end of the module chain.

Reply:
This if fast mode. I like it. Marking resolved.